src/index.html

```diff
@@ -1,14 +1,39 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Enclosures</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="outer" id="qa-block">
+      <section class="outer__section">
+        <p class="inner">
+          In programming languages, a closure is a technique for implementing lexically scoped name binding in a
+          language with first-class functions. Operationally, a closure is a record storing a function together with an environment.
+        </p>
+      </section>
+    </div>
+    <div class="interval"></div>
```

Comment:
Using empty elements with margin for layout management is a bad practice.
Please, avoid it in the next tasks.
Add margins to one of adjacent elements.